packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -187,7 +187,9 @@ function prepareTextNode(node, key, text, createNode, createNodeId) {
 function prepareJSONNode(node, key, content, createNodeId, i = ``) {
   const str = JSON.stringify(content)
   const JSONNode = {
-    ...content,
+    ...(typeof content === 'string' || content instanceof String
```

Comment:
can we invert this to test if `content` is object (`_.isPlainObject(content)`)? this way it would handle other types than just strings (like numbers and booleans)